src/mocks/handlers.ts

```diff
@@ -1,33 +1,66 @@
 import { rest } from "msw";
 
+export const resourceToMockMap = {
+  POKE_LIST: {
+    // msw specific pattern
+    url: "https://pokeapi.co/api/v2/pokemon",
+    // cypress url pattern
+    urlPattern: "https://pokeapi.co/api/v2/pokemon",
+    payload: {
+      count: 1279,
+      next: "https://pokeapi.co/api/v2/pokemon?offset=20&limit=20",
+      previous: null,
+      results: [
+        { name: "mike", url: "https://pokeapi.co/api/v2/pokemon/1/" },
+        { name: "alex", url: "https://pokeapi.co/api/v2/pokemon/2/" },
+        { name: "lily", url: "https://pokeapi.co/api/v2/pokemon/3/" },
+        { name: "larry", url: "https://pokeapi.co/api/v2/pokemon/4/" },
+      ],
+    },
+  },
+  POKE_BY_ID: {
+    url: "https://pokeapi.co/api/v2/pokemon/:id/",
+    urlPattern: "https://pokeapi.co/api/v2/pokemon/*",
+    payload: {
+      sprites: {
+        front_default:
+          "https://raw.githubusercontent.com/PokeAPI/sprites/master/sprites/pokemon/1.png",
+      },
+    },
+  },
+} as const;
+
 export const handlers = [
-  // // Uncomment to see requests mock at a global level
-  // rest.get("https://pokeapi.co/api/v2/pokemon", async (req, res, ctx) => {
-  //   console.log("req", req.url);
-  //   return res(
-  //     ctx.status(200),
-  //     ctx.json({
-  //       count: 1279,
-  //       next: "https://pokeapi.co/api/v2/pokemon?offset=20&limit=20",
-  //       previous: null,
-  //       results: [
-  //         { name: "mike", url: "https://pokeapi.co/api/v2/pokemon/1/" },
-  //         { name: "alex", url: "https://pokeapi.co/api/v2/pokemon/2/" },
-  //         { name: "lily", url: "https://pokeapi.co/api/v2/pokemon/3/" },
-  //         { name: "larry", url: "https://pokeapi.co/api/v2/pokemon/4/" },
-  //       ],
-  //     })
-  //   );
-  // }),
-  // rest.get("https://pokeapi.co/api/v2/pokemon/:id/", (req, res, ctx) => {
-  //   return res(
-  //     ctx.status(200),
-  //     ctx.json({
-  //       sprites: {
-  //         front_default:
-  //           "https://raw.githubusercontent.com/PokeAPI/sprites/master/sprites/pokemon/1.png",
-  //       },
-  //     })
-  //   );
-  // }),
+  rest.get(resourceToMockMap["POKE_LIST"].url, async (req, res, ctx) => {
+    const mockHeader = req.headers.get("x-mock");
```

Comment:
Set in`cy.mock` [here](https://github.com/mcMickJuice/msw-mock-poc/pull/1/files#diff-6c107e2390c5ad84e123f3be09d7b91ba35de8f8cc550d1f358c75806df47645R36)